blockchain/source/contracts/Market.sol

```diff
@@ -699,6 +633,89 @@ contract Market is Ownable {
     }
     // INTERNAL
 
+    function InteralBill(uint dealID) internal returns (bool){
+        require(deals[dealID].status == DealStatus.STATUS_ACCEPTED);
```

Comment:
its more applicable at `Bill`

Reply:
why? Is it valid to run IntarnalBill on closed deal?